share/revng/revng-check-conventions-README.md

```diff
@@ -0,0 +1,48 @@
+# Information about `revng-check-conventions`
```

Comment:
Move to `doc/revng-check-conventions.md`.
One day we'll make a `man` out of it or something.

Reply:
Moved